tasks/main.yml

```diff
@@ -9,6 +9,13 @@
   tags:
     - install
 
+- name: "Add patch for redis-5.0.5 in Ubuntu 22.04"
+  ansible.builtin.lineinfile:
+    path: /usr/local/src/redis-5.0.5/src/sds.h
+    regexp: '^const char \*SDS_NOINIT;'
+    line: 'extern const char *SDS_NOINIT;'
+  when: redis_version == '5.0.5'
```

Comment:
In the task title this task specific for ubuntu 22, dont we need to put that condition here also @rishavOneForAll?